tests/test_scripts.py

```diff
@@ -358,14 +361,39 @@ def test_transfer_learning(tmpdir: str) -> None:
         named_configs=["cartpole"] + ALGO_FAST_CONFIGS["rl"],
         config_updates=dict(
             common=dict(log_dir=log_dir_data),
-            reward_type="RewardNet_shaped",
+            reward_type="RewardNet_unshaped",
```

Comment:
This started throwing errors once I added validation to ensure that reward functions where actually shaped when they were loaded as shaped. I think this is supposed to be unshaped since it's loading the test reward function. Is that correct? Or is this supposed to be a shaped reward?

Reply:
I think this is right, good catch!